src/development/platform-integration/web/initialization.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Customizing web app initialization
+description: Customize how Flutter apps are initialized on the web
+---
+
+# Customizing web app initialization
+
+You can customize how a Flutter app is initialized on the web
+using the `_flutter.loader` JavaScript API.
+This API can be used to display a loading indicator in CSS,
+prevent the app from loading based on a condition,
+or wait until the user presses a button before showing the app.
+
+The initialization process is split into the following stages:
+
+* **Loading the entrypoint script** - 
+    Fetches the `main.dart.js` script
+    and initializes the service worker.
+* **Initializing the Flutter engine** -
+    Initializes Flutter's web engine
+    by downloading required resources
+    such as assets, fonts, and CanvasKit.
+* **Running the app** -
+    Prepares the DOM for your Flutter app and runs it.
+
+This page shows how to add custom behavior
+at each stage of the initialization process.
+
+## Getting started
+
+By default, the `index.html` file
+generated by the `flutter create` command
+contains a script tag
+that calls `loadEntrypoint` from the `flutter.js` file:
+
+```
+<html>
+   <head>
+      <!-- ... -->
+      <script src="flutter.js" defer></script>
+   </head>
+   <body>
+      <script>
+         window.addEventListener('load', function (ev) {
+             // Download main.dart.js
+             _flutter.loader.loadEntrypoint({
+                 serviceWorker: {
+                     serviceWorkerVersion: serviceWorkerVersion,
+                 }
+             }).then(function (engineInitializer) {
+                 // Initialize the Flutter engine
+                 return engineInitializer.initializeEngine();
+             }).then(function (appRunner) {
+                 // Run the app
+                 return appRunner.runApp();
+             });
+         });
+      </script>
+   </body>
+</html>
+```
+
+
+
+{{site.alert.note}}
+*Note:* In Flutter 2.10 or earlier,
+this script doesn't support customization.
+To upgrade your index.html file to the latest version,
+see Upgrading an older project.
+{{site.alert.end}}
```

Comment:
- Removes extra "Note" which already exists with `site.alert.note`
- Adds code font to `index.html` to be consistent with rest of file
- Adds link to mentioned section

```suggestion
{{site.alert.note}}
  In Flutter 2.10 or earlier,
  this script doesn't support customization.
  To upgrade your `index.html` file to the latest version,
  see [Upgrading an older project](#upgrading-an-older-project).
{{site.alert.end}}
```